app/src/main/java/org/mozilla/fenix/wallpapers/Wallpaper.kt

```diff
@@ -51,6 +49,13 @@ enum class WallpaperOrigin {
     REMOTE,
 }
 
+val Wallpaper.drawableId: Int get() = when (name) {
+    "amethyst" -> R.drawable.amethyst
+    "cerulean" -> R.drawable.cerulean
+    "sunrise" -> R.drawable.sunrise
+    else -> -1
+}
+
```

Comment:
I think this will only apply to Firefox wallpapers, right? Having different types for different wallpapers could make this a bit simpler to handle related https://github.com/mozilla-mobile/fenix/pull/23505/commits/730a7f7e1db1349a0823c84a34375bec4412ffb3#r796792287

Reply:
This way we could have different properties that make sense for specific wallpaper types

Reply:
Oh yeah, thanks for the reminder! I'll make some changes around that

Reply:
We could evaluate that in follow up :)